eth_tester/utils/backend_testing.py

```diff
@@ -238,6 +238,105 @@ def test_get_nonce(self, eth_tester):
         assert nonce >= UINT256_MIN
         assert nonce <= UINT256_MAX
 
+    #
+    # Fee History
+    #
+    @pytest.mark.parametrize(
+        "block_count,newest_block,reward_percentiles,expected",
+        [
+            [
+                3,
+                "latest",
+                [],
+                {
+                    "base_fee_per_gas": [300657803, 343608917, 392695905],
+                    "gas_used_ratio": [0.0, 0.0, 0.0],
+                    "reward": [],
+                },
+            ],
+            [
+                1,
+                "safe",
+                [],
+                {
+                    "base_fee_per_gas": [300657803],
+                    "gas_used_ratio": [0.0],
+                    "reward": [],
+                },
+            ],
+            [
+                1,
+                "finalized",
+                [],
+                {
+                    "base_fee_per_gas": [300657803],
+                    "gas_used_ratio": [0.0],
+                    "reward": [],
+                },
+            ],
+            [
+                1,
+                "earliest",
+                [],
+                {
+                    "base_fee_per_gas": [],
+                    "gas_used_ratio": [],
+                    "reward": [],
+                },
+            ],
+        ],
```

Comment:
If we agree to add "pending" as a valid API value as per the comment above, we should add the test case here.